model/post.go

```diff
@@ -238,6 +238,17 @@ type GetPostsSinceOptions struct {
 	SortAscending            bool
 }
 
+type GetPostsSinceForSyncOptions struct {
+	ChannelId       string
+	Since           int64 // exclusive
+	Until           int64 // inclusive
```

Comment:
I am not sure I understand. From the queries both `Since` and `Until` seem to be inclusive, first one is doing `UpdateAt >= ts` while the other `UpdateAt <= ts`.

Reply:
Good catch. The comments were left over from when NextSyncAt was LastSyncAt which needed UpdateAt>ts.

Reply:
Fixed via https://github.com/mattermost/mattermost-server/pull/16962/commits/3f4dc12f54db024764ab8b953b8d947174283d18